src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectwizard/viewmodel/ProjectWizardViewModel.kt

```diff
@@ -52,6 +55,17 @@ class ProjectWizardViewModel : ViewModel() {
         }
     }
 
+    private fun filterToAvailableLanguges() {
+        collectionRepo
+            .getRootSources()
+            .observeOnFx()
+            .subscribe { retrieved ->
+                retrieved.forEach {collection ->
+                    filteredLanguages.addAll(languages.filter { it.id == collection.resourceContainer?.language?.id })
+                }
```

Comment:
This looks like it'll create duplicate entries in filteredLanguages if you have multiple resource containers in the same language. I suggest you make a set of languages from the RCs, and use that set to filter languages.